### PR TITLE
Change tool package version to 1.0.0-preview1

### PR DESCRIPTION
**Pull request by pranavkm**
Additionally fixing some project.json warnings

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @muratg \ @natemcmaster

File: makefile.shade [hunk 1]
```diff
@@ -2,7 +2,7 @@
 var VERSION='0.1'
 var FULL_VERSION='0.1'
 var AUTHORS='Microsoft Open Technologies, Inc.'
-
+-BuildQuality = '';
```
Comment (natemcmaster) [inline makefile.shade]: What's this?

**Comment by natemcmaster** [comment]
Two issues from travis CI:
```
fail: /home/travis/build/aspnet/EntityFramework/src/Microsoft.EntityFrameworkCore.Tools/project.json(80,19): warning DOTNET1015: The 'packInclude' option is deprecated. Use 'files' in 'packOptions' instead.
```
```
info: Producing nuget package "Microsoft.EntityFrameworkCore.Tools.1.0.0-preview1-t160506190044" for Microsoft.EntityFrameworkCore.Tools
fail: NuGetResources.SemVerSpecialVersionTooLong
```

File: makefile.shade (hunk 1 above)
Comment (pranavkm) [inline makefile.shade]: It allows us to specify a build quality on a per project basis.

**Comment by pranavkm** [comment]
@natemcmaster - switching to `packOptions` has issues which is the reason I'm leaving it as is for now (https://github.com/dotnet/cli/issues/2829). I pushed a change to KoreBuild to fix the exception and kicked it off again.

**Comment by pranavkm** [comment]
@natemcmaster rebuilding fixed Travis.

**Comment by natemcmaster** [comment]
Assuming https://github.com/aspnet/KoreBuild/pull/71 goes in soon, :shipit:

**Pull request merged**